build.gradle

```diff
@@ -57,6 +57,11 @@ def copyFiles(def projectPath, def exclusions) {
             include '**/*.meta'
             into "/LoadingScreen/Textures"
         }
+        from('GooglePlayInstant/LoadingScreen/Scripts')
```

Comment:
Should we just remove `'**/*.meta'` from the `exclude 'Samples', '**/*.meta'` line 53? Then we don't need to add back meta files manually?